AuxTel/AuxTel-Troubleshooting/ATCS-Troubleshooting/ATMCS-fails-enable.rst

```diff
@@ -0,0 +1,65 @@
+.. Include one Primary Author and list of Contributors (comma separated) between the asterisks (*):
+.. |author| replace:: *Paulina-Venegas-S.*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *Karla-Aubel*, *Ioana-Soutela*
+
+.. This is the label that can be used as for cross referencing this procedure.
+.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
+.. _AuxTel-Mount-Control-System-Fails-to-Enable:
+.. Each section should includes a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+.. _E-Stop-Procedure: https://obs-ops.lsst.io/AuxTel/AuxTel-Non-standard-Operations/ATCS-Non-standard-Procedures/E-Stop-Procedure.html
+
+###############################################################
+AuxTel Mount Control System Fails to Enable / E-Stop is Engaged
+###############################################################
+
+.. _AuxTel-Mount-Control-System-Fails-to-Enable-Procedure-Overview:
+
+Overview
+========
+
+When attempting to enable ATCS from the ATScriptQueue with the script **auxtel/enable_atcs.py**, ATMCS fails to transition to enable.
+
+.. _AuxTel-Mount-Control-System-Fails-to-Enable-Procedure-Error-Diagnosis:
+
+Error diagnosis
+===============
+
+ATMCS fails to enable with the rest of the ATCS with the SAL script **auxtel/enable_atcs.py**.
+
+
+.. error::
+
+    Fault event in ATMCS while in enable state (port). Nasmyth 1 drive fault bit is ON
+
```

Comment:
I suggest to include the traceback as code output:

**..code-block:: text
:caption: Error:**
Fault event in ATMCS while in enable state (port). Nasmyth 1 drive fault bit is ON